ironfish-cli/src/commands/miners/start.ts

```diff
@@ -94,6 +104,7 @@ export class Miner extends IronfishCommand {
         batchSize,
         host: host,
         port: port,
+        minerName,
```

Comment:
We don't need to make a new variable when we only want to pass the flag through here

```suggestion
        minerName: flags.name,
```